elements/pf-alert/demo/custom-icon.html

```diff
@@ -0,0 +1,39 @@
+<div class="demo-description">
+    <section class="alerts-page">
+        <pf-alert class="a" state="cogear">
+            <h3 slot="header">Success alert title</h3>
+        </pf-alert>
+        <pf-alert class="b" state="cogear">
+            <h3 slot="header">Success alert title</h3>
+        </pf-alert>
+    </section>
+</div>
```

Comment:
```suggestion
<div class="alert-demo">
  <pf-alert class="a" state="cogear">
    <h3 slot="header">Success alert title</h3>
  </pf-alert>
  <pf-alert class="b" state="cogear">
    <h3 slot="header">Success alert title</h3>
  </pf-alert>
</div>
```

try to avoid wrapping demos with extra HTML elements